docs/docs-next/docs/concepts/automation.md

```diff
@@ -1 +1,66 @@
-# Automation
\ No newline at end of file
+---
+title: About Automation
+---
+
+Dagster has support for several types of automation. All automation in Dagster responds to some external event.
```

Comment:
"All automation in Dagster responds to some external event" isn't really true. I would maybe strike this paragraph and replace with a single sentence

> There are several ways to automate the execution of your data pipelines with Dagster.